File: AlphaParking.Java.AuthService/javaauth/src/main/java/com/auth/event_bus_utils/integration_events/UserEditedIntegrationEvent.java
package com.auth.event_bus_utils.integration_events;

import com.auth.models.User;

public class UserEditedIntegrationEvent extends IntegrationEvent {
    private User updatedUser;
    private User oldUser;

    public UserEditedIntegrationEvent(User updatedUser, User oldUser) {
        this.setUpdatedUser(updatedUser);
        this.setOldUser(oldUser);
    }

    /**
     * @return the oldUser
     */
    public User getOldUser() {
        return oldUser;
    }

    /**
     * @param oldUser the oldUser to set
     */
    public void setOldUser(User oldUser) {
        this.oldUser = oldUser;
    }

    /**
     * @return the updatedUser
     */
    public User getUpdatedUser() {
        return updatedUser;
    }

    /**
     * @param updatedUser the updatedUser to set
     */
    public void setUpdatedUser(User updatedUser) {
        this.updatedUser = updatedUser;
    }

}

File: AlphaParking/AlphaParking.BLL/EventBus/IntegrationEvents/UserEditedIntegrationEvent.cs
using AlphaParking.BLL.EventBus.Abstractions;
using AlphaParking.Models;

namespace AlphaParking.BLL.EventBus.IntegrationEvents
{
    public class UserEditedIntegrationEvent: IntegrationEvent
    {
        public User OldUser { get; set; }
        public User UpdatedUser { get; set; }
        

        public UserEditedIntegrationEvent(User oldUser, User updatedUser) 
        {
            OldUser = oldUser;
            UpdatedUser = updatedUser;
        }
    }
}